### PR TITLE
Small edit in input topics

### PR DESCRIPTION
**Pull request by ganicke**

### CODE REVIEW

File: libs/thermometer/docs/reference/input/on-temperature-condition-changed.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# on Temperature Condition Changed
+# on Temperate Condition Changed
```
Comment (pelikhan) [inline libs/thermometer/docs/reference/input/on-temperature-condition-changed.md]: temperate?

Comment (ganicke) [inline libs/thermometer/docs/reference/input/on-temperature-condition-changed.md]: That's what you called it:

void onTemperateConditionChanged(TemperatureCondition condition, int temperature, TemperatureUnit unit, Action handler)

**Comment by pelikhan** [comment]
That's a typo :)

**Comment by ganicke** [comment]
void onWarmingCondition() {}

**Comment by ganicke** [comment]
'onTemperatureConditionChanged' has returned.

**Pull request merged**